packages/common/src/ui/icons/ArrowRight.tsx

```diff
@@ -2,9 +2,8 @@ import React from 'react';
 import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
 
 export const ArrowRightIcon = (props: SvgIconProps): JSX.Element => {
-  const combinedProps: SvgIconProps = { color: 'primary', ...props };
   return (
-    <SvgIcon {...combinedProps} viewBox="0 0 16 16">
+    <SvgIcon {...props} viewBox="0 0 16 16">
```

Comment:
The buttons will correctly set the colour now.. but they won't if we set a default colour.. so I thought to just pass through props and let them do it themselves now

Reply:
Removing the default colour has made the Svg icon story go strange, they were defaulting to primary, and now only some are.
Could you add a primary prop to the `Primary` of that story and maybe add another for 'Default' or something, so show what happens without providing a colour?